Library/Homebrew/requirements/maximum_macos_requirement.rb

```diff
@@ -12,7 +12,11 @@ def initialize(tags)
 
   def message
     <<-EOS.undent
-      OS X #{@version.pretty_name} or older is required.
+      This formula either does not compile or function as expected on newer
```

Comment:
My former English teacher will shoot me out of a cannon for this, but I couldn't remember if the standard negative `nor` / `or` usage applied in the context where one may not be valid, i.e. some formula break on compile but some formula compile fine but break at runtime. It's not universally the case that a formula would both break at compile *and* at runtime.